<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="jajit-z01" name="selectorshub.Healeniumfirst" tests="1" failures="0" timestamp="2022-04-20T20:21:21 IST" time="102.479" errors="1">
  <testcase name="healniumtest" time="102.479" classname="selectorshub.Healeniumfirst">
    <error type="org.openqa.selenium.NoSuchElementException" message="Failed to find element using By.xpath: //a[text()=&#039;Gmail&#039;]
For documentation on this error, please visit: https://selenium.dev/exceptions/#no_such_element
Build info: version: &#039;4.0.0-beta-4&#039;, revision: &#039;29f46d02dd&#039;
System info: host: &#039;jajit-z01&#039;, ip: &#039;10.104.53.200&#039;, os.name: &#039;Windows 10&#039;, os.arch: &#039;amd64&#039;, os.version: &#039;10.0&#039;, java.version: &#039;1.8.0_161&#039;
Driver info: driver.version: unknown">
      <![CDATA[org.openqa.selenium.NoSuchElementException: Failed to find element using By.xpath: //a[text()='Gmail']
For documentation on this error, please visit: https://selenium.dev/exceptions/#no_such_element
Build info: version: '4.0.0-beta-4', revision: '29f46d02dd'
System info: host: 'jajit-z01', ip: '10.104.53.200', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_161'
Driver info: driver.version: unknown
at com.epam.healenium.handlers.proxy.BaseHandler.findElement(BaseHandler.java:67)
at com.epam.healenium.handlers.proxy.SelfHealingProxyInvocationHandler.invoke(SelfHealingProxyInvocationHandler.java:39)
at com.sun.proxy.$Proxy9.findElement(Unknown Source)
at selectorshub.Healeniumfirst.healniumtest(Healeniumfirst.java:27)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: java.lang.NullPointerException
at com.epam.healenium.processor.HealingProcessor.validate(HealingProcessor.java:22)
at com.epam.healenium.processor.BaseProcessor.process(BaseProcessor.java:40)
at com.epam.healenium.processor.BaseProcessor.process(BaseProcessor.java:49)
at com.epam.healenium.processor.BaseProcessor.process(BaseProcessor.java:49)
at com.epam.healenium.handlers.proxy.BaseHandler.findElement(BaseHandler.java:58)
... 31 more
]]>
    </error>
  </testcase> <!-- healniumtest -->
</testsuite> <!-- selectorshub.Healeniumfirst -->
